pwndbg/config.py

```diff
@@ -135,30 +184,56 @@ def __str__(self):
     def __bool__(self):
         return bool(self.value)
 
+    # Comparing operaters
+    # Ref: http://portingguide.readthedocs.io/en/latest/comparisons.html
+    # If other is Parameter, comparing by optname. Used in `sorted` in `config` command.
+    # Otherwise, compare `self.value` with `other`
+    def __eq__(self, other):
+        if isinstance(other, gdb.Parameter):
+            return self.optname == other.optname
+        else:
+            return self.value == other
+
     def __lt__(self, other):
-        return self.optname <= other.optname
+        if isinstance(other, gdb.Parameter):
+            return self.optname < other.optname
+        else:
+            return self.value < other
 
-    def __div__(self, other):
-        return self.value / other
+    # Operaters
+    def __add__(self, other):
+        return self.value + other
 
-    def __floordiv__(self, other):
-        return self.value // other
+    def __radd__(self, other):
```

Comment:
I don't find a good base class to do that. Our use case is weird because we just want to proxy all operator to `self.value`. People often inherit the type of `value` to do that in this use case. However, we can't do that because the type of `value` is dynamic and reply on gdb's implementation. That's why we need a operator proxy to wrap the operators.

I think this use case is limited and even we create a abstract class for this, it will be only one class will use it. At least, until now we only have one class that need to implement operator. What do you think?